tools/dhtnode.cpp

```diff
@@ -29,7 +29,7 @@ extern "C" {
 using namespace dht;
 
 void print_usage() {
-    std::cout << "Usage: dhtnode [-p local_port] [-b bootstrap_host:port]" << std::endl << std::endl;
+    std::cout << "Usage: dhtnode [-h] [-v [logfile]] [-i] [-d] [-n network_id] [-p local_port[:port]] [-b [bootstrap_host]]" << std::endl << std::endl;
```

Comment:
The syntax suggests "-h" can be used along with other options, which is not the case (printing the help message is a special case that makes other flags meaningless).

If it was to be listed (up to your choice), it should have it's own "Usage:" line, like what you did for the manpage.